openlibrary/plugins/openlibrary/js/Browser.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Convert url paramaters to an object
+ * @param {String} urlSearch everything after (and including) the '?' of the url
+ * @returns {{ [param: string]: string }}
+ */
+export function getJsonFromUrl(urlSearch) {
+    const query = urlSearch.substr(1);
+    const result = {};
+    if (query) {
+        query.split('&').forEach(part => {
+            const item = part.split('=');
+            result[item[0]] = decodeURIComponent(item[1]);
+        });
+    }
+    return result;
+}
+
+/**
+ * @param {String} url
+ * @param {String} parameter name of param to remove
+ * @returns {String}
+ */
+export function removeURLParameter(url, parameter) {
+    var urlparts = url.split('?');
+    var prefix = urlparts[0];
+    var query, paramPrefix, params, i;
+    if (urlparts.length >= 2) {
+        query = urlparts[1];
+        paramPrefix = `${encodeURIComponent(parameter)}=`;
+        params = query.split(/[&;]/g);
```

Comment:
@jdlrobson : how does this differ from the split in the getJsonFromUrl function.

Reply:
Hmmm, it seems to split at semicolons as well (for some reason)? 🤔